driver/pace/driver/run.py

```diff
@@ -1,6 +1,8 @@
 import abc
 import dataclasses
 import functools
+import os
+import subprocess
```

Comment:
I generally try to avoid subprocess for something as simple as getting a git hash (just because it is somewhat finicky). Why did you opt for it to fetch it from here compared to as an input arg?

Reply:
Because I asked for it. What's finicky about subprocess?